### DISCUSSION TITLE
Calendar Reopen

### DISCUSSION THREAD

**Discussion by soap47** [fomantic/Fomantic-UI#2216] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
I don't know if this issue is a bug but it can be annoying sometimes.
If you select a date from the datepicker it will pick that date but stay focused on the date field so if for any reason you need to change that date clicking the field again won't open the datepicker you have to click outside and click back in the field to reopen the datepicker, 
You can see this behavior in the demo page too (https://fomantic-ui.com/collections/form.html#calendar)

Is there a fix for that?

**Comment by lubber-de** [comment, marked as answer]
That has been fixed in the nightly beta by 
- https://github.com/fomantic/Fomantic-UI/pull/2002 and
- https://github.com/fomantic/Fomantic-UI/pull/1508

**Comment by soap47** [comment]
Thank you
